src/main/resources/META-INF/plugin.xml

```diff
@@ -64,6 +64,7 @@
 
     <change-notes><![CDATA[
       <ul>
+        <li>10.3 - Support Jakarta namespace. 9 new C++ MISRA 2023 rules. Remove support for Node.JS 14. Support TypeScript 5.3. 5 new rules for JavaScript.</li>
```

Comment:
About:
> Remove support for Node.JS 14

I have several questions:
* last time SonarJS dropped support for a version, they also deprecated the next one (you can see below in the release notes). Is it also the case here?
* What is the minimal version now? I think we have it documented in several places, including the [docs website](https://docs.sonarsource.com/sonarlint/intellij/getting-started/requirements/#jstscss-analysis), so you might need to check with Kevin